exercises/practice/hamming/.docs/instructions.append.md

```diff
@@ -0,0 +1,14 @@
+# Implementation notes: exceptions
+
+This exercise expects the code to raise exceptions in certain circumstances.
+Exceptions are usually raised when an operation cannot be completed for some
+reason and there is no value that can be returned which would be correct.
+Instead, an exception is raised. When raising an exception, the tests expect
+that the exception class in instantiated and contains a message.
+
+```python
+# Avoid this. There is no message and the tests will fail:
+raise ValueError
+# Do this. Provide a clear message with as much detail as possible:
```

Comment:
```suggestion
# Do this. Provide a clear and actionable message:
```

I don't think we always want to go into detail, but whatever we do, there must be "just" enough information to take an action. A large assssss error message which you can't solve isn't as good as a short, solvable, error. Right?

Reply:
Changed.